src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateAction.java

```diff
@@ -0,0 +1,39 @@
+package org.csanchez.jenkins.plugins.kubernetes;
+
+import hudson.Extension;
+import hudson.model.Action;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import jenkins.model.TransientActionFactory;
+
+@Extension
+public class PodTemplateAction extends TransientActionFactory<KubernetesCloud> implements Action{
```

Comment:
Usual idiom is to have the `Action` class, and a static inner class annotated with `@Extension` for the factory.

Reply:
Thanks for pointing out. I have fixed it in https://github.com/jenkinsci/kubernetes-plugin/pull/1443/commits/c86ad898917fb7018a087ba8f4be3c3f38bf3bba